### PR TITLE
properly skip datastore tests from other repos

### PR DESCRIPTION
**Pull request by bshaffer**
note: this is from my fork, to test it works

### CODE REVIEW

File: appengine/flexible/datastore/tests/LocalTest.php [hunk 1]
```diff
@@ -25,6 +25,9 @@ public function setUp()
         if (!getenv('GCLOUD_PROJECT')) {
             $this->markTestSkipped('Must set GCLOUD_PROJECT');
         }
+        if (!getenv('GOOGLE_APPLICATION_CREDENTIALS')) {
```
Comment (tmatsuo) [inline appengine/flexible/datastore/tests/LocalTest.php]: `GOOGLE_APPLICATION_CREDENTIALS` is actually set in the `.travis.yaml` so it doesn't skip. I think you need to check the file existence.

**Comment by tmatsuo** [comment]
`GOOGLE_APPLICATION_CREDENTIALS` should be set somewhere if you want to remove it from `.travis.yaml` anyways. I think it's more appropriate to check the file existance.

**Comment by bshaffer** [comment]
@tmatsuo I added `GOOGLE_APPLICATION_CREDENTIALS` to the repo settings' environment variables. 

Also I disagree, I think checking if the file exists is a pain because it's an additional check.

**Comment by bshaffer** [comment]
We could possibly add the `SkipIfMissingCredentials` trait to the testing repo and then add that to all the tests.

**Comment by bshaffer** [comment]
Closing for now, we'll find a better way to do this.

**Pull request closed without merging**